pennylane/_device.py

```diff
@@ -406,6 +407,11 @@ def execute(self, queue, observables, parameters={}, **kwargs):
         self._parameters.update(parameters)
 
         results = []
+        if self._shot_vector is not None:
+            warnings.warn(
+                "Specifying a list of shots is only supported for"
```

Comment:
Oh, there's a space missing here.
```suggestion
                "Specifying a list of shots is only supported for "
```